frontstage/templates/secure-messages-draft.html

```diff
@@ -46,8 +46,7 @@ <h1 class="saturn">Edit a draft message</h1>
                                 class="secure-message-form__subject input input--text"
                                 type="text"
                                 name="secure-message-subject"
-                                value="{{draft['subject']}}"
-                                required/>
+                                value="{{draft['subject']}}"/>
```

Comment:
Should this be `{{ draft['subject'] }}`?

Should probably at least keep it consistent with the other changes.